components/Chat.tsx

```diff
@@ -1,13 +1,67 @@
-import React, { Component } from 'react'
-import { Text, View } from 'react-native'
+import React from 'react'
+import { Text, View, TouchableOpacity, Image, StyleSheet } from 'react-native'
 
-const Chat = props => {
-  //Create markup for the individual chat in the chatlist
+interface Props {
+  chat: Chat
+}
+
+interface Chat {
+  chat_id: number,
+  participants: Array<Participant>
+}
+
+interface Participant {
+  email: string,
+  first_name: string,
+  profile_photo: string
+}
+
+const Chat: React.FunctionComponent<Props> = props => {
   return (
-    <View>
-      <Text> This is the chat Component </Text>
-    </View>
+    <TouchableOpacity style={styles.chat}>
+      {props.chat.participants.map(participant => {
```

Comment:
As it's possible for a chat to contain more participants it's necessary to loop through them to show all profile pictures and names. This needs to be done within the component, so we only render one chat, but with multiple participants